spec/classes/puppet_agent_windows_install_spec.rb

```diff
@@ -44,26 +44,14 @@
 
         let(:facts) { facts.merge({:is_pe => true}) }
 
-        if Puppet.version < '4.0.0'
```

Comment:
Not sure why this entire test is removed.  If it's a Puppet 3 thing, might be best to add this as a separate commit

Reply:
Our team's decided to remove Puppet 3 tests as a part of releasing the puppet-agent module 2.0. See https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/350